### ISSUE TITLE
notifyItemMoved method

### ISSUE THREAD

**Issue by XxGoliathusxX** [bignerdranch/expandable-recycler-view#98]
Hey,
im looking for an alternative method for 

notifyItemMoved(fromPosition, toPosition);

I tried:

 notifyParentItemChanged(fromPosition);
            notifyParentItemChanged(toPosition);

But it didnt work. Any solutions?

**Comment by paul-turner** [comment]
Right now we haven't made the Parent/Child moved notify methods. The best I can suggest is a poor mans moving:
```
remove the parentItem
notifyParentItemRemoved
add the parentItem to new position
notify ParentItemInserted
```
Note, `notifyParentItemChanged` does not allow any "structural changes" which moving a parent from one point to another would be.

We do have plans for creating notifyMoved methods that will work, I'll make a separate issue for those to keep better track soon.

**Comment by XxGoliathusxX** [comment]
Hmm,
notifyItemMoved(fromPosition, toPosition) 

for parent-items works. How do i swap two child items?

Object listItem = getListItem(i);
                if(listItem instanceof ParentWrapper){
                   // DO PARENT STUFF
                }else{
                    //????
                }

I cant wrap the listItem into a ChildWrapper or Childobject. 

So lastParent.getChildItemList().add(//CHILDOBJECT NEEDED) 

doesnt work.

**Comment by paul-turner** [comment]
This may have already been figured out but in
`lastParent.getChildItemList().add(//CHILDOBJECT NEEDED)`
the child object that you need is in fact the `listItem` Object variable you pulled out.

Not sure if that helps any.

**Comment by dgreenhalgh** [comment]
Resolved in 2.1.0.
